examples/pytorch/text-generator/README.md

```diff
@@ -150,148 +142,31 @@ local   text-generator   live     1            1           17m           3.1285
 
 The output above indicates that one replica of your API was requested and is available to serve predictions. Cortex will automatically launch more replicas if the load increases and will spin down replicas if there is unused capacity.
 
-Show additional information for your API (e.g. its endpoint) using `cortex get <api_name>`:
-
-```bash
-$ cortex get text-generator --env aws
-
-status   up-to-date   requested   last update   avg request   2XX
-live     1            1           17m           -             -
-
-metrics dashboard: https://us-west-2.console.aws.amazon.com/cloudwatch/home#dashboards:name=cortex
-endpoint: https://***.execute-api.us-west-2.amazonaws.com/text-generator
-...
-```
-
-Use your new endpoint to make requests to your API on AWS:
-
-```bash
-$ curl https://***.execute-api.us-west-2.amazonaws.com/text-generator \
-    -X POST -H "Content-Type: application/json" \
-    -d '{"text": "machine learning is"}'
-
-"machine learning is ..."
-```
-
-<br>
-
-## Perform a rolling update
-
-When you make a change to your `predictor.py` or your `cortex.yaml`, you can update your api by re-running `cortex deploy`.
+## Run on GPUs
 
-Let's modify `predictor.py` to set the length of the generated text based on a query parameter:
+If your cortex cluster is using GPU instances (configured during cluster creation), you can run your text generator API on GPUs. Add the `compute` field to your API configuration and re-run `deploy`:
 
 ```python
-# predictor.py
-
-import torch
-from transformers import GPT2Tokenizer, GPT2LMHeadModel
-
-
-class PythonPredictor:
-    def __init__(self, config):
-        self.device = "cuda" if torch.cuda.is_available() else "cpu"
-        print(f"using device: {self.device}")
-        self.tokenizer = GPT2Tokenizer.from_pretrained("gpt2")
-        self.model = GPT2LMHeadModel.from_pretrained("gpt2").to(self.device)
-
-    def predict(self, payload, query_params):  # this line is updated
-        input_length = len(payload["text"].split())
-        output_length = int(query_params.get("length", 20))  # this line is added
-        tokens = self.tokenizer.encode(payload["text"], return_tensors="pt").to(self.device)
-        prediction = self.model.generate(tokens, max_length=input_length + output_length, do_sample=True)  # this line is updated
-        return self.tokenizer.decode(prediction[0])
-```
-
-Run `cortex deploy` to perform a rolling update of your API:
-
-```bash
-$ cortex deploy --env aws
+api_spec = {
+  'name': 'text-generator',
```

Comment:
nit: double quotes throughout (unless there is a specific reason to using single quotes, we might as well keep it consistent with the code in our examples)